### DISCUSSION TITLE
GIF support

### DISCUSSION THREAD

**Discussion by Cardiacs1964** [TheQwertiest/foo_spider_monkey_panel#191] [Q&A] [answered]
You can implement GIF support into Spider Monkey Panel ( WebP )?

Best Regards

Cardiacs

![](https://user-images.githubusercontent.com/112690467/195619531-c484c91c-d670-4e00-b84e-c6b8bfb971af.gif)

**Comment by TheQwertiest** [comment, marked as answer]
See https://github.com/TheQwertiest/foo_spider_monkey_panel/issues/214 for the answer (tldr: can't be implemented)
